hazelcast-sql/src/main/java/com/hazelcast/jet/sql/impl/validate/HazelcastSqlValidator.java

```diff
@@ -469,6 +469,9 @@ public Void visit(SqlIdentifier id) {
                 // not every identifier is a table
                 if (table != null) {
                     HazelcastTable hazelcastTable = table.unwrap(HazelcastTable.class);
+                    if (hazelcastTable.getTarget() instanceof ViewTable) {
```

Comment:
This is wrong, we can't just ignore views. For example, this query is supposed to fail, but it doesn't:
```sql
CREATE VIEW v AS SELECT * FROM TABLE(GENERATE_STREAM(10));
select v.v from v order by 1; -- should fail
```

Reply:
Add isStream cached property, then will be reworked as part of #20032

Reply:
Wouldn't it be better to move the check after view expansion where we have correct, up-to-date information?